mongorestore/oplog.go

```diff
@@ -210,6 +210,14 @@ func (restore *MongoRestore) HandleNonTxnOp(oplogCtx *oplogContext, op db.Oplog)
 		return fmt.Errorf("error filtering UUIDs from oplog: %v", err)
 	}
 
+	// The h field was removed in 7.1.0-rc0 as part of SERVER-69062
+	if restore.serverVersion.GTE(db.Version{7, 0, 0}) {
```

Comment:
Based on your comments elsewhere I _suspect_ it won't matter, but is it safe to strip the `h` fields even for 7.0 patch versions? This will do so for 7.0.1, for example.

Reply:
Yeah I wasn't 100% sure how 7.1.0-rc0 would evaluate so I just did this since it doesn't matter.